client/src/components/arabicListWords/style.css

```diff
@@ -0,0 +1,44 @@
+.word1 {
+  position: absolute;
```

Comment:
Why are you using absolute positioning here? Check this on all of them.

Reply:
for CSS I want to work it today on my home.

Reply:
Okay, I don't want to merge it like this though. Can you at least delete the absolute positioning?